tools/schema/prerequisites.json

```diff
@@ -209,6 +209,21 @@
                 }
             }
         },
+        "zone_properties": {
+            "propertyOrder": 1,
+            "title": "zone_properties",
+            "description": "Checks that the player is in a zone with the specified properties. Properties may be inverted by starting them with `!`. Click Properties, then type in the layer name in the input box and click the 'add' button. Layer and property names are case sensitive.",
```

Comment:
Maybe add a comment here that the default layer name is "default" - which is what people probably should be testing against?